### PR TITLE
Blowout uses battlestorm visual effect

### PR DESCRIPTION
**Pull request by flappybatpal**
[QOL]

## About the PR 
Uses the tint effect from battle storms to show which areas of a radiation blowout are unsafe.

## Why's this needed? 
Radiation blowouts are currently a slightly confusing event for players. Blowouts have no visible indication of unsafe areas and radiation was recently changed to not show status effect icon, making it an invisible killer. New players can be overwhelmed by a 40 second countdown, alarms and no clear understanding of what locations are safe or why they are taking damage.
Some of the safe/not safe areas are surprising due to how areas work, this will now be completely clear to players.
Players also often leave early before radiation has reduced to a safe level. It will now be more obvious there is still radiation.

image

## Changelog

```changelog
(u)Flappybat
(+)Radiation blowout uses the battlestorm effect to show unsafe areas.
```

### CODE REVIEW

**Comment by mathis650** [comment]
yes please

**Comment by Gannets** [comment]
This is one of those new player traps that's been around forever and nobody's had the heart to do anything about. Pleased to see the change even if I'll occasionally miss the schadenfreude of watching assistants dying in the hallways.

File: code/modules/events/blowout.dm [hunk 1]
```diff
@@ -88,6 +89,7 @@
 					break
 				if (!A.permarads)
 					A.irradiated = 0
+				A.icon_state = ""
```
Comment (Sovexe) [inline code/modules/events/blowout.dm:92]:
```suggestion
				A.icon_state = null
```

Comment (flappybatpal) [reply code/modules/events/blowout.dm]: BTW this was copied from battlestorm so it will be wrong there too

**Review by Sovexe — APPROVED** [review]

**Review by ZeWaka — APPROVED** [review]

**Comment by pali6** [comment]
I personally feel like the battlestorm effect is a bit too intense and would prefere something slightly toned down. But a good change either way.

**Review by ZeWaka — APPROVED** [review]

File: code/modules/events/blowout.dm [hunk 2]
```diff
@@ -21,7 +21,7 @@
 		SPAWN_DBG(0)
 			for_by_tcl(A, /obj/machinery/door/airlock)
 				LAGCHECK(LAG_LOW)
-				if (A.z != 1)
+				if (A.z != Z_LEVEL_STATION)
 					break
```
Comment (ZeWaka) [inline code/modules/events/blowout.dm:25]:
```suggestion
					continue
```

**Pull request merged**